packages/thirdweb/src/bridge/Status.ts

```diff
@@ -0,0 +1,146 @@
+import type { Hex as ox__Hex } from "ox";
+import type { ThirdwebClient } from "../client/client.js";
+import { getClientFetch } from "../utils/fetch.js";
+import { UNIVERSAL_BRIDGE_URL } from "./constants.js";
+import type { Status } from "./types/Status.js";
+
+/**
+ * Retrieves a Universal Bridge quote for the provided sell intent. The quote will specify the expected `destinationAmount` that will be received in exchange for the specified `originAmount`, which is specified with the `sellAmountWei` option.
+ *
++ * The returned status will include both the origin and destination transactions and any finalized amounts for the route.
+ *
+ * @example
+ * ```typescript
+ * import { Bridge } from "thirdweb";
+ *
+ * const status = await Bridge.status({
+ *   transactionHash: "0xe199ef82a0b6215221536e18ec512813c1aa10b4f5ed0d4dfdfcd703578da56d",
+ *   chainId: 8453,
+ *   client: thirdwebClient,
+ * });
+ * ```
+ *
+ * If the transaction is complete, a response might look like:
+ * ```typescript
+ * {
+ *   status: 'completed',
+ *   originAmount: 200000000000000n,
+ *   destinationAmount: 188625148000000n,
+ *   originChainId: 8453,
+ *   destinationChainId: 2741,
+ *   originTokenAddress: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
+ *   destinationTokenAddress: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
+ *   transactions: [
+ *     {
+ *       chainId: 8453,
+ *       transactionHash: '0xe199ef82a0b6215221536e18ec512813c1aa10b4f5ed0d4dfdfcd703578da56d'
+ *     },
+ *     {
+ *       chainId: 2741,
+ *       transactionHash: '0xa70a82f42330f54be95a542e1fcfe6ed2dd9f07fb8c82ae67afb4342319f7433'
+ *     }
+ *   ]
+ * }
+ * ```
+ *
+ * If the transaction is still pending, a response might look like:
+ * ```typescript
+ * {
+ *   status: "pending",
+ *   originAmount: 1000000000000000000n,
+ *   originChainId: 466,
+ *   destinationChainId: 1,
+ *   originTokenAddress: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+ *   destinationTokenAddress: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+ *   transactions: [
+ *     {
+ *       transactionHash: "0xe199ef82a0b6215221536e18ec512813c1aa10b4f5ed0d4dfdfcd703578da56d",
+ *       chainId: 466,
+ *     }
+ *   ]
+ * }
+ * ```
+ *
+ * If the transaction failed, a response might look like:
+ * ```typescript
+ * {
+ *   status: "failed",
+ *   transactions: [
+ *     {
+ *       transactionHash: "0xe199ef82a0b6215221536e18ec512813c1aa10b4f5ed0d4dfdfcd703578da56d",
+ *       chainId: 466,
+ *     }
+ *   ]
+ * }
+ * ```
+ *
+ * This status is for a **single origin transaction only**. If your route involves multiple transactions, you'll need to get the status for each of them individually.
+ *
+ * If sending multiple dependent sequential transactions, wait until `status` returns `completed` before sending the next transaction.
+ *
+ * You can access this function's input and output types with `status.Options` and `status.Result`, respectively.
+ *
```

Comment:
would either create a new tag for these docs, or use `@buyCrypto` which is the tag we use for the other pay apis.

prob a new tag makes sense? we should also deprecate the old ones btw and point to the new ones